spacy_llm/models/hf/base.py

```diff
@@ -48,34 +52,45 @@ def __call__(self, prompts: Iterable[_PromptType]) -> Iterable[_ResponseType]:
         RETURNS (Iterable[_ResponseType]): API responses.
         """
 
-    @property
+    def _check_model(self) -> None:
+        """Checks whether model is supported. Raises if it isn't."""
+        if (
+            self._name not in self.get_model_names()
+            and self._name.replace(f"{self.get_hf_account()}/", "")
+            not in self.get_model_names()
```

Comment:
At this point, shouldn't the name always include the prefix? So why the two checks?
If we feel this gets too confusing, we can also consider storing two variables internally, the `hf_account` and the `model_name` and never merge them up until we actually use the request to the HF API.

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/176/commits/8d6ae3fa2c6246ad18554862fe586a6a3c1c2b28.